creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -238,15 +238,68 @@ spec:
     pod: {}
 ----
 
+______________________________________________________________________________
+**Note:** If a specific MAC address is configured in the virtual machine
+interface spec, it's passed to underlying CNI plugin that should configure
+backend to allow for this particular address. While some plugins are known to
+work without special support for custom MAC addresses, others do (for example,
+OpenShift SDN needs to configure OVS flows to allow traffic meant for the
+chosen MAC address).
+______________________________________________________________________________
+
+______________________________________________________________________________
+**Note:** If a plugin doesn't support `mac` attribute defined by Kubernetes
+Network Plumbing Working Group, one can use `tuning` plugin that is one of
+reference plugins maintained by the group. This can be used as follows:
+
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: ptp-mac
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
+      "name": "ptp-mac",
+      "plugins": [
+        {
+          "type": "ptp",
+          "ipam": {
+            "type": "host-local",
+            "subnet": "10.1.1.0/24"
+          },
+        },
+        {
+          "type": "tuning",
+        }
+      ]
+    }'
+----
+
+
+This approach won't work for other plugins that needs special treatment for
+custom MAC addresses. For example, OpenShift SDN is not compatible with
+`tuning` plugin.
+
+Plugins that handle custom MAC addresses natively: `ovs`.
```

Comment:
We do? I think you need to use tuning with OVS CNI.

Reply:
@phoracek what is this code for then? https://github.com/kubevirt/ovs-cni/blob/50c2abbc4209ef07fcce8fb41c044c24df571690/pkg/plugin/plugin.go#L135-L154

Reply:
:man_facepalming: forgot about this one

Reply:
it's ok, it was 3 months ago :)